app.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "thing",
+  "scripts": {},
+  "env": {
+    "CERTIFICATE": {
+      "required": true
+    },
+    "HEROKU_APP_NAME": {
```

Comment:
Hmmm. I thought `required: true` copied the value from the staging app which would not be desirable. Maybe I don't get it.

Reply:
HEROKU_APP_NAME follows a different pattern, which is weird: https://devcenter.heroku.com/articles/github-integration-review-apps

Reply:
Ah, weird, but okay :)